ui/components/app/nft-details/nft-details.js

```diff
@@ -202,42 +200,44 @@ export default function NftDetails({ nft }) {
             justifyContent={JustifyContent.spaceBetween}
           >
             <div>
-              <Typography
+              <Text
                 color={TextColor.textDefault}
-                variant={TypographyVariant.H4}
-                fontWeight={FONT_WEIGHT.BOLD}
-                boxProps={{ margin: 0, marginBottom: 2 }}
+                variant={TextVariant.headingSm}
+                as="h4"
+                marginBottom={2}
```

Comment:
```suggestion
                marginBottom={2}
                marginTop={0}
                marginLeft={0}
                marginRight={0}
```

Reply:
@georgewrmarshall I'm curious if there's a default value when we don't pass in any for margin, maybe i'm wrong in this one.

Reply:
So the `Typography` component had some default margin values set which in this instance were being removed via the `boxProps` the `Text` component doesn't have any default margins set so they are already the equivalent of `0`

Reply:
Gotcha. Thanks for explaining. Will resolve this since it’s not valid.